packages/graphql/src/translate/translate-read.ts

```diff
@@ -196,8 +196,8 @@ function translateRootField({
 
     const cypher: string[] = [
         subStr.matchAndWhereStr,
-        subStr.projectionSubqueries,
         ...(sortOffsetLimit.length > 1 ? sortOffsetLimit : []),
+        subStr.projectionSubqueries,
```

Comment:
This is all it was in the end?! 😆 At least it was an easy fix!

Reply:
Not only that, this seems to have fixed other related issues with `@cypher`